### PR TITLE
Apply chapter-specific icons.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/401497359171917

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://41-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://41-bb-docs.surge.sh

**Comment by bbewan** [comment]
SVGs are inlined, so we can use CSS for hover states. That required some tweaking of the SVGs, to apply classes instead of specific fill/stroke colors.

File: _img/icon-integrations.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="11" viewBox="0 0 16 11">
```
Comment (benedict-jw) [inline _img/icon-integrations.svg]: @bbewan - the integrations icon doesn't seem to have exported properly. One of the arrows appears to be outlined, as opposed to filled in. I checked Zeplin and tried exporting it there and it looks fine to me, so I wonder if there are other styles being applied to the icon? Let me know how I can help debug.

**Comment by bbdocs** [comment]
See the website here: http://41-bb-docs.surge.sh

**Comment by dennispi** [comment]
Yesssssss

**Pull request merged**